dvc/tree/ssh/__init__.py

```diff
@@ -246,8 +246,7 @@ def getsize(self, path_info):
             return ssh.getsize(path_info.path)
 
     def _download(self, from_info, to_file, name=None, no_progress_bar=False):
-        assert from_info.isin(self.path_info)
-        with self.ssh(self.path_info) as ssh:
+        with self.ssh(from_info) as ssh:
```

Comment:
We do have `import-url` func tests, but the credentials situation is not trivial there, so we have to use a remote:// notation, which mitigates this bug. Could add two unit tests for these specific places in code, but not sure if it is really worth it for these leftovers.

Reply:
Ok, leaving as is in the interest of time.